qiskit_nature/second_q/algorithms/excited_states_solvers/excited_states_eigensolver.py

```diff
@@ -16,7 +16,7 @@
 
 import logging
 
-from qiskit.algorithms.eigensolvers import Eigensolver
+from qiskit_algorithms.eigensolvers import Eigensolver
```

Comment:
```suggestion
from qiskit_algorithms import Eigensolver
```
While you can import from the folders, as you would have to do in qiskit.algorithms since it still had the QI based ones imported as conveniences at that level, in qiskit_algorithms this can be dropped. I would argue its better to use the suggestion as algos got reorganized in the past and that always worked. And if you look it up in the docs its at that place too.